setup/setup.sh

```diff
@@ -0,0 +1,88 @@
+USERNAME="student"
+MINIKUBE_DIR=$HOME"/.minikube"
+CERT_DIRECTORY="$MINIKUBE_DIR/redhat-certs"
+SCRIPTPATH="$( cd -- "$(dirname "$0")" >/dev/null 2>&1 ; pwd -P )"
+
+function create_certificates() {
+    echo "Creating certificates..."
+    if [ ! -d $CERT_DIRECTORY ]; then
+        mkdir $CERT_DIRECTORY
+    fi
+
+    cd $CERT_DIRECTORY
+    openssl genrsa -out $USERNAME.key 2048 &> /dev/null
+    openssl req -new -key $USERNAME.key -out $USERNAME.csr -subj "/CN=$USERNAME/O=group1" &> /dev/null
+    openssl x509 -req -in $USERNAME.csr -CA $MINIKUBE_DIR/ca.crt -CAkey $MINIKUBE_DIR/ca.key -CAcreateserial -out $USERNAME.crt -days 500 &> /dev/null
+}
+
+function create_namespace() {
+    echo "Creating namespace 'redhat-test'..."
```

Comment:
This namespace looks weird.
To align with Developer Sandbox we need to create something like `username-dev` and `username-stage`.

Reply:
+1